wger/core/views/user.py

```diff
@@ -48,100 +43,92 @@
     PasswordConfirmationForm,
     RegistrationForm,
     RegistrationFormNoCaptcha,
-    UserLoginForm)
-from wger.core.models import Language
-from wger.manager.models import (
-    WorkoutLog,
-    WorkoutSession,
-    Workout
+    UserLoginForm,
 )
+from wger.core.models import Language
+from wger.manager.models import WorkoutLog, WorkoutSession, Workout
```

Comment:
Revert to the previous import style. Don't put all imported constants on the same line.